packages/xml-views-validation/api.d.ts

```diff
@@ -93,3 +102,60 @@ export interface NonUniqueIDIssue extends BaseUI5XMLViewIssue {
 export interface NonStableIDIssue extends BaseUI5XMLViewIssue {
   kind: "NonStableIDIssue";
 }
+
+export function validateUnknownEnumValue(
```

Comment:
This public API is a little verbose.
It makes me think we want to expose all these validators
inside some object? and sub properties?

Reply:
we discussed this on voice